.github/workflows/render-site.yaml

```
@@ -0,0 +1,44 @@
+name: Run quarto render
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, edited, ready_for_review]
```

```suggestion
    types: [opened, synchronize, edited, ready_for_review]
```

You don't need the label function here. In fact, given the new CI check takes several minutes to run, we should minimize how often we trigger.